.travis.yml

```diff
@@ -36,6 +37,7 @@ env:
    - CCACHE_VERSION=3.3.1
    - CMAKE_VERSION=3.7.2
    - MASON="$(pwd)/scripts/mason.sh"
+   - NODE="4"
```

Comment:
I need the values above to install `yarn` for me. I think it also causes node 4 and 6 to be in the nvm cache. That seems to work at least for non OSX builds which need an explicit `nvm install` step.